README.md

```diff
@@ -1,8 +1,755 @@
+</p>
+
+### Features
+
+<p>
+   On top of the regular Symfony skeleton, we have made opinionated choices and configured your project
+   to get you started faster and make you more productive:
+   <ul>
+       <li>
+           The <a href="https://github.com/symfony/webapp-pack/blob/main/composer.json">webapp pack</a>, which includes all recommended Symfony packages for web applications, has been installed and configured, including:
+
+  <ul>
+      <li><a href="http://twig.symfony.com/" target="_blank">Twig</a> as a templating system</li>
+      <li><a href="https://symfony.com/doc/current/doctrine.html" target="_blank">Doctrine</a> for the ORM and database migrations</li>
+      <li><a href="https://symfony.com/doc/current/frontend.html#webpack-encore" target="_blank">Webpack Encore</a> for JavaScript management</li>
+      <li><a href="https://symfony.com/doc/current/frontend/ux.html" target="_blank">Symfony UX</a> for progressive JavaScript applications</li>
+      <li><a href="https://symfony.com/bundles/SymfonyMakerBundle/current/index.html" target="_blank">Maker bundle</a> to let you quickly generate code for your controllers, migrations, and more</li>
+      <li><a href="https://symfony.com/doc/current/reference/configuration/web_profiler.html" target="_blank">Symfony Web Profiler</a> to ease debugging issues locally.</li>
+  </ul>
+       </li>
+       <li>A configuration optimized for <strong>PostgreSQL as a database</strong>.</li>
+       <li>A <strong>Docker compose configuration</strong> for PostgreSQL and a <strong>mail catcher</strong> for local development.</li>
+       <li>An <strong>optimized Platform.sh configuration</strong> for Symfony and all included services.</li>
+       <li>A bootstrapped <strong>database migration system</strong>.</li>
+       <li>An optimized <strong>async configuration</strong> for sending emails and notifications asynchronously with retry and failure features enabled on PostgreSQL.</li>
+   </ul>
+</p>
+
+## Getting started
+
+### Deploy
+
+#### Quickstart
+
+
+The quickest way to deploy this template on Platform.sh is by clicking the button below.
+This will automatically create a new project and initialize the repository for you.
 
 <p align="center">
 <a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/symfonycorp/platformsh-symfony-template-metadata/main/template-metadata-sf6.0-php8.1-webapp.yaml&utm_content=symfonycorp&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+    <img src="https://platform.sh/images/deploy/deploy-button-lg-blue.svg" alt="Deploy on Platform.sh" width="175px">
 </a>
 </p>
+<br/>
+
+
+You can also quickly recreate this project locally with the following command:
+
+```bash
+#TODO @FabienPotencier add this repo to Packagist 
```

Comment:
Hi @fabpot could you please add this repo to packagist to let user create project using the console?